packages/l10n/data/countries/es.json

```diff
@@ -149,7 +149,7 @@
   "mf": "San Martín",
   "mg": "Madagascar",
   "mh": "Islas Marshall",
-  "mk": "Macedonia del Norte",
+  "mk": "Macedonia",
```

Comment:
can we revert this?

Reply:
That's the data we get from cldr 🤔

Reply:
Yes but it's *wrong*. [ref](https://www.theguardian.com/world/2019/feb/12/nato-flag-raised-ahead-of-north-macedonias-prospective-accession)

Reply:
So does it mean that we can't run the script anymore, or that we need to manually patch things up? Is that the only _wrong_ thing that you found or are there are cases?

Reply:
Fixed in https://github.com/commercetools/merchant-center-application-kit/pull/511/commits/1a92f79cb3ee8798cdb77b20ba077e49ce47cba6

Reply:
PS: apparently not all languages have the latest translations (e.g. for "north macedonia") 🤷‍♂️

Reply:
I added the translations in a previous PR for north macedonia.

If everyone is fine with it, I guess we can revert to the "wrong" countries and wait until CLDR catches up 🤷‍♂️ I tried to find a repo on which to log an  issue but I couldn't find one, is it some mailing list? Do you know how to contact them about their data being outdated?

Reply:
I'd rather prefer to use the official list instead of patching some names manually. They eventually catch up, so I wouldn't worry much about it.

Reply:
Can you look into contacting them about the outdated countries?

Reply:
I found this ticket related to the macedonia translations (from 7 days ago): https://unicode.org/cldr/trac/changeset/14975

I suppose that those will be available in the next release.